storage/Utils/SystemCmd.cc

```diff
@@ -368,17 +416,20 @@ namespace storage
 
 	do
 	{
-	    y2deb("[0] id:" <<  _pfds[0].fd << " ev:" << hex << (unsigned)_pfds[0].events << dec << " [1] fs:" <<
-		  (_combineOutput?-1:_pfds[1].fd) << " ev:" << hex << (_combineOutput?0:(unsigned)_pfds[1].events));
-	    int sel = poll( _pfds, _combineOutput?1:2, 1000 );
+	    y2deb("[0] id:" <<	_pfds[1].fd << " ev:" << hex << (unsigned)_pfds[1].events << dec << " [1] fs:" <<
+		  (_combineOutput?-1:_pfds[2].fd) << " ev:" << hex << (_combineOutput?0:(unsigned)_pfds[2].events));
+	    int sel = poll( _pfds, _combineOutput?2:3, 1000 );
```

Comment:
Well, yes, I completely agree; I'd do that for new code. But the rationale here was to keep the diff minimalistic.